database-plugins/src/main/java/co/cask/hydrator/plugin/DBManager.java

```diff
@@ -47,9 +47,6 @@ public void validateJDBCPluginPipeline(PipelineConfigurer pipelineConfigurer, St
     Preconditions.checkArgument(!(dbConfig.user == null && dbConfig.password != null),
                                 "user is null. Please provide both user name and password if database requires " +
                                   "authentication. If not, please remove password and retry.");
```

Comment:
looks like the username is always required, why can't we make that field non-nullable ?